test/test_framework.py

```diff
@@ -856,9 +927,26 @@ def test_mutable_types(self):
         class SomeObject(Object):
             state = StoredState()
 
+        class WrappedFramework(Framework):
+            def __init__(self, framework):
+                object.__setattr__(self, '_framework', framework)
```

Comment:
This seems like it's mixing has-a and is-a.  Seems like it would be cleaner to not inherit from `Framework` since you're proxying everything anyway.

Reply:
You have to inherit from Framework, because we have code that does isinstance(blah, Framework)

I original did not inherit, but it doesn't work. And going "full framework" doesn't work because you have to wrap one that we create. I suppose I could write something like create_framework that returns a wrapped on instead.

Reply:
That said, it actually was much clearer to not wrap the object, and just pass in __init__ the params we needed. So I pushed an update to do that.
I'm not a huge fan of __getattr__ style programming, but I was thinking I needed to wrap the Framework we were already creating. But actually we can just instantiate it in a better way.